app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,14 @@
+class LikesController < ApplicationController
+  def create
+    @like = Like.new
+    @like.author = current_user
+    @like.post = Post.find(params[:post_id])
+    if @like.save
+      flash[:notice] = 'Like added successfully!'
+      redirect_to user_post_path(user_id: params[:user_id], id: params[:post_id])
+    else
+      flash[:alert] = "Couln't add like!"
+      render :new, status: :unprocessable_entity
+    end
+  end
```

Comment:
## [OPTIONAL]
- [ ] I think the project might look better if the user can just like once a post.